classes/services/queryBuilders/PKPSubmissionQueryBuilder.inc.php

```diff
@@ -277,13 +277,18 @@ public function getQuery() {
 			$locale = \AppLocale::getLocale();
 			$this->columns[] = Capsule::raw('COALESCE(publication_tlps.setting_value, publication_tlpsl.setting_value)');
 			$q->leftJoin('publications as publication_tlp', 's.current_publication_id', '=', 'publication_tlp.publication_id')
-				->leftJoin('publication_settings as publication_tlps', 'publication_tlp.publication_id', '=', 'publication_tlps.publication_id')
-				->where('publication_tlps.setting_name', '=', 'title')
-				->where('publication_tlps.locale', '=', $locale);
+				->leftJoin('publication_settings as publication_tlps', function($join) use ($locale) {
+					$join->on('publication_tlp.publication_id', '=', 'publication_tlps.publication_id');
+					$join->on('publication_tlps.setting_name', '=', Capsule::raw("'title'"));
```

Comment:
The Laravel framework will take care of escaping the values, so you can pass just 'title' here. The same applies for the lines below.

Reply:
We're not using the raw method to escape the 'title', but instead, to add a clause to the join's 'on' clause

Reply:
That's right! :)

Reply:
Sorry for butting in here when it's already being capably reviewed :) but I think the preferred syntax would be `->where` instead of `->on`; since you're calling it on the `$join` object it'll be included in the `ON` SQL clause, but because it's a `where` instead of an `on` you'll be able to remove the `Capsule::raw` wrapper.

Reply:
A crazy thing: 
When we use `->on` we cannot use just `'title'` the error `Unknown column 'title' in 'on clause' ` occurs -- the 'title' is not provided as bindings parameter but in the code...
When we however use `->where` we can only use `'title'` -- the 'title' is provided as bindings parameter...
So yes, use `->where` and only `'title'`...